### ISSUE TITLE
Include power tower tag as a poi

### ISSUE THREAD

**Issue by fgeragb** [osmandapp/Osmand#9398]
Hello,
After requesting support they recommended me to open an issue.
We are using OsmAnd Android and we would like to include in the list of custom POIs the element Power Tower:
https://wiki.openstreetmap.org/wiki/Tag%3Apower%3Dtower
When we search in custom filter we find water tower and tower, and related to electric power we find power plant and power generator, but not Power Tower.
Is it possible to configure OsmAnd to include power tower in the list of custom POIs?
Attached are screens with the elements we found (can't find Power tower).
Best Regards

**Comment by vshcherb** [comment]
OsmAnd maps don't store that information as POI, we have limited set of objects. It is possible to create a custom maps though we don't plan to include many point objects as POI.

**Comment by fgeragb** [comment]
Thanks vshcherb,
I don't really want it in the default POI list, but in the custom list if it is possible.
When you go to the top left icon search > categories > custom search (at the bottom of the categories list) > There is a lot of POI categories here, and the question is if is it possible to include it here.
There is also an User defined category, but I don't know how to include another custom POI.
Thanks a lot, Gerardo

**Comment by vshcherb** [comment]
Nope, you need to build custom map using OsmAndMapCreator and also you need to modify poi_types.xml and add this object https://github.com/osmandapp/OsmAnd-resources/blob/master/poi/poi_types.xml.
With new map put into OsmAnd folder you will be able to create custom filter and even search type

**Comment by NotSoImportant** [comment]
Is that the same reason why I can create power pole pois but cannot edit them once uploaded?
